src/createServer.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { router: expensesRouter } = require('./routes/expenses.router');
+const { router: usersRouter } = require('./routes/users.router');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const server = express();
+
+  server.use('/users', express.json(), usersRouter);
+  server.use('/expenses', express.json(), expensesRouter);
```

Comment:
[error]  should return empty array if no users
[error] should return empty array if no expenses
[checklist] Data should be empty initially. 

You can create function that will clear all your data when the server will start e.g.: `usersServices.removeAllUsers()`